app/konnect/dev-portal/access-and-approval/sso.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Configure SSO for Dev Portal
+content_type: how-to
+---
+
+You can configure single sign-on (SSO) for {{site.konnect_short_name}} Dev Portal with OIDC. This allows developers to log in to Dev Portals by using their SSO credentials. 
+
+<!--Do developers get auto approved in Dev Portal then?-->
```

Comment:
Developers do get auto-approved by Konnect when using SSO. This is because we outsource the approval process to the IDP instance when using SSO. Meaning it is possible to restrict who can sign up via the IDP rather than through Konnect.